pkg/controllers/management/eks/eks_cluster_handler.go

```diff
@@ -635,3 +651,22 @@ func notFound(err error) bool {
 	}
 	return false
 }
+
+// generateValuesYaml generates a YAML string containing any
+// necessary values to override defaults in values.yaml. If
+// no defaults need to be overwritten, an empty string will
+// be returned.
+func generateValuesYaml() (string, error) {
+	values := eksOperatorValues{
+		HTTPProxy:  os.Getenv("HTTP_PROXY"),
+		HTTPSProxy: os.Getenv("HTTPS_PROXY"),
+		NoProxy:    os.Getenv("NO_PROXY"),
+	}
+
+	valuesYaml, err := yaml.Marshal(values)
+	if err != nil {
+		return "", err
+	}
+
+	return string(valuesYaml), nil
```

Comment:
nit: could just return `yaml.Marshal`, same thing.

Reply:
At some point it needs to be converted from []byte to string. Otherwise, that would be the same.